symbols/overlay29.yml

```diff
@@ -6014,7 +6025,9 @@ overlay29:
         NA: 0x2321104
         JP: 0x23225B0
       description: |-
-        Checks that the given monster is standing on a tile it can stand on given its movement type, and warps it to a random location if it's not.
+        Checks that the given monster is standing on a tile it can stand on given its
+        movement type. If the monster is a non-leader ally, they will be warped to the
+        leader. Otherwise, the monster is warped randomly.
```

Comment:
This new description no longer states that the warping only takes place if the monster cannot stand on the tile